main/.vitepress/config.mjs

```diff
@@ -2,18 +2,31 @@ import { defineConfig } from 'vitepress';
 import { nav } from './themeConfig/nav.js';
 import { rewrites } from './themeConfig/rewrites.js';
 
+const SITE_TITLE = 'Agoric Documentation';
+const SITE_DESCRIPTION =
+  'The blockchain framework tailored for JavaScript developers.';
```

Comment:
I don't have any objections to this copy, but I don't think I'm a good person to approve it.
@sufyaankhan ?